src/Build.OM.UnitTests/Construction/ConstructionEditing_Tests.cs

```diff
@@ -40,13 +40,13 @@ public void AddTargetConvenience()
   <Target Name=""t"" />
 </Project>");
 
-            Assert.Equal(true, project.HasUnsavedChanges);
+            Assert.True(project.HasUnsavedChanges);
```

Comment:
Was this whole commit suggested fixes, or did you have to do anything by hand?

Reply:
All suggested fixes.